pkg/storage/s3/s3.go

```diff
@@ -322,7 +337,10 @@ func (d *driver) CreateStorage(cr *imageregistryv1.Config, modified *bool) error
 				util.UpdateCondition(cr, imageregistryv1.StorageEncrypted, operatorapi.ConditionFalse, "Unknown Error Occurred", err.Error(), modified)
 			}
 		} else {
-			util.UpdateCondition(cr, imageregistryv1.StorageEncrypted, operatorapi.ConditionTrue, "Encryption Successful", "Default encryption was successfully enabled on the S3 bucket", modified)
+			util.UpdateCondition(cr, imageregistryv1.StorageEncrypted, operatorapi.ConditionTrue, "Encryption Successful", fmt.Sprintf("Default %s encryption was successfully enabled on the S3 bucket", encryptionType), modified)
+			d.Config.Encrypt = true
+			cr.Status.Storage.S3 = d.Config.DeepCopy()
+			cr.Spec.Storage.S3 = d.Config.DeepCopy()
```

Comment:
per our chat, this needs to be done even in cases when we don't manage the bucket, otherwise we'll think the storage has changed everytime we re-sync, so we'll keep churning through this logic.  (not quite a hotloop,but unnecessary).

Reply:
updated in next push